[2024-04-13 17:07:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:19] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:34460 failed checkpoint posthandshake: Connected to self
[2024-04-13 17:07:19] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 17:07:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:49] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 3023af312ae4d24d [::1]:38998 failed checkpoint posthandshake: Connected to self
[2024-04-13 17:07:49] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 3023af312ae4d24d [::1]:20134 failed checkpoint posthandshake: Connected to self
[2024-04-13 17:07:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-04-13 17:07:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
